README.md

```diff
@@ -163,27 +192,23 @@ Each OpenJDK variant has a canonical branch that is built:
 Our Jenkins [Pipelines](https://ci.adoptopenjdk.net/view/Pipelines/) ([source code](https://github.com/AdoptOpenJDK/openjdk-build/tree/master/pipelines)) build
 binaries and then push them through a test pipeline (**NOTE** Test pipeline is skipped for certain unstable combinations).
 
-**NOTE:** Once the Test jobs have stabilised, the pipeline described below will change so that only tested binaries are released to
-Nightly and Release repos for consumption by the Website and/Or API.  See [The Quality Bar Discussion](https://github.com/AdoptOpenJDK/openjdk-tests/issues/186)
+**NOTE:** Once the Test jobs have stabilised, the pipeline described below will change so that only tested binaries are released to Nightly and Release repositories for consumption by the Website and/Or API.  See [The Quality Bar Discussion](https://github.com/AdoptOpenJDK/openjdk-tests/issues/186)
 for details.
 
 #### 3.1 Build
 
-[Builds](https://ci.adoptopenjdk.net/) are run on the
-[Supported Platforms](https://docs.google.com/spreadsheets/d/1T_sYpMe1y2cSdiP83HA2VWnTH6B-Nbl3ppss0gEpACc/edit?usp=sharing). The Jenkins leader
-sends the build jobs to the Jenkins followers based on a tagging system configured in the Jenkins jobs, e.g. `centos6&&x64&&build`. See
-[openjdk-build](https://github.com/AdoptOpenJDK/openjdk-build) and [openjdk-infrastructure](https://github.com/AdoptOpenJDK/openjdk-infrastructure) for details.
+[Builds](https://ci.adoptopenjdk.net/) are run on the [Supported Platforms](https://docs.google.com/spreadsheets/d/1T_sYpMe1y2cSdiP83HA2VWnTH6B-Nbl3ppss0gEpACc/edit?usp=sharing). The Jenkins leader
```

Comment:
That document only has up to JDK10 so sugget using the link to that information on our website
```suggestion
[Builds](https://ci.adoptopenjdk.net/) are run on the [Supported Platforms](https://adoptopenjdk.net/supported_platforms.html). The Jenkins leader
```